classes/class-wc-connect-api-client.php

```diff
@@ -483,6 +504,9 @@ protected function request_headers() {
 			$headers[ 'Content-Type' ] = 'application/json; charset=utf-8';
 			$headers[ 'Accept' ] = 'application/vnd.woocommerce-connect.v' . static::API_VERSION;
 			$headers[ 'Authorization' ] = $authorization;
+
+			$wc_helper_auth_info = WC_Connect_Functions::get_wc_helper_auth_info();
+			$headers[ 'X-Woo-Signature' ] = $this->request_signature_wccom( $wc_helper_auth_info['access_token_secret'] );
```

Comment:
Should there be an is_wp_error check here as well?

Reply:
Good point. I think I avoided it at first because I was worried about preventing requests that did _not_ need the Helper auth info. But I don't see a case where we wouldn't want the auth info.